### PR TITLE
Enable disable sections admin merch

### PR DESCRIPTION
**Pull request by amsalmeron**
Here I created sections for the enabled and disables merchants for the admin merchant index page.

Note -  I set the merchant status to be enabled by default. Also had to add the status attribute to get some of the tests working.

### CODE REVIEW

File: app/views/admin_merchants/index.html.erb [hunk 1]
```diff
@@ -10,19 +10,23 @@
...
 
-<h2>List of All Merchants:</h2>
-<% @merchants.each do |merchant| %>
-  <div id="merchants-<%= merchant.id %>">
+<h2>All Merchants:</h2>
 
-  <h3>-<%= link_to "#{merchant.name}", "/admin/merchants/#{merchant.id}" %></h3>
+<h3>Enabled Merchants</h3>
+<% @merchants.each do |merchant| %>
+<div id="merchants-<%= merchant.id %>">
```
Comment (B-gann21) [inline app/views/admin_merchants/index.html.erb]: This view page looks pretty good, one thing i would do is enclose the whole 'Enabled Merchants' section (header and each block included) inside of a div tag, with id="enabled-merchants". This way you can use a within block in the test, to test that only merchants with a status of 'enabled' can be found in the 'enabled merchants' section. 
The same can be done with the disabled merchants section

File: spec/features/admin/merchants/index_spec.rb [hunk 2]
```diff
@@ -240,6 +237,19 @@
...
+        merchant_2 = Merchant.create!(name: 'Antonio', status: 0)
+        merchant_3 = Merchant.create!(name: 'Chris', status: 1)
+        merchant_4 = Merchant.create!(name: 'Craig', status: 1)
+
+        visit "/admin/merchants"
+        expect(page).to have_content("Enabled Merchants")
+        expect(page).to have_content("Disabled Merchants")
+        expect(merchant_4.name).to appear_before(merchant_1.name)
+        expect(merchant_3.name).to appear_before(merchant_2.name)
+      end
```
Comment (B-gann21) [inline spec/features/admin/merchants/index_spec.rb:262]: This test is where we could utilize a within block: `within '#enabled-merchants' do`; inside of that block we would expect the page to have only enabled merchants, and to_not have disabled merchants. 

The opposite can be done in a `within '#disabled-merchants' do` block

**Review by B-gann21 — CHANGES_REQUESTED** [review]
Looks good overall. Check in-line comments for notes on using <div> tags and `within` blocks 👌 Should be a quick refactor and then I'd call it ready to merge

File: app/views/admin_merchants/index.html.erb (hunk 1 above)
Comment (amsalmeron) [reply app/views/admin_merchants/index.html.erb]: There are now within block to test the enable/disable sections on the admin merchant index page.

**Review by B-gann21 — APPROVED** [review]

**Pull request merged**